ci-operator/step-registry/openstack/test/dpdk/openstack-test-dpdk-commands.sh

```diff
@@ -145,4 +145,7 @@ else
     exit 1
 fi
 
-echo "Successfully ran NFV tests"
+oc delete pod "${CNF_POD}" -n "${CNF_NAMESPACE}"
+oc delete namespace "${CNF_NAMESPACE}"
```

Comment:
Nit: deleting the Namespace already deletes the Pod inside of it. Looks like line 148 is not needed

Reply:
Same applies on openstack-test-sriov-commands.sh